pkg/volume/gce_pd/gce_util.go

```diff
@@ -146,7 +146,7 @@ func (gceutil *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner) (volum
 		return "", 0, err
 	}
 
-	err = cloud.CreateDisk(name, zone.FailureDomain, int64(requestGB))
+	err = cloud.CreateDisk(name, zone.FailureDomain, int64(requestGB), *c.options.CloudTags)
```

Comment:
Is's a good idea, I'll implement it as separate PR (and also for AWS + OpenStack/Cinder).